server/src/main/java/org/apache/iotdb/db/service/MetricsServiceMBean.java

```diff
@@ -22,8 +22,6 @@
 
 public interface MetricsServiceMBean {
```

Comment:
Move it to package org.apache.iotdb.db.service.metrics;

Reply:
ok, I will fix this.